lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -18,6 +18,11 @@ admin_state_ethernet_noswitchport_shutdown:
     /N7K/:
       default_value: "shutdown"
 
+all_channel_groups:
+  multiple:
+  config_get: "show running interface all | include channel-group"
```

Comment:
You don't need this config_get. You can inherit the config_get from the _template entry and it provides the same data; by using a different command it forces another call to nxapi for every interface on the node.